api/access/invoker.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package access
+
+import "github.com/onflow/flow-go/model/flow"
```

Comment:
I prefer putting this into brackets immediately for the same reasons that Go adds a trailing comma, so that a change only creates one line of diff instead of 3 :grin:

Reply:
Me too, but Goland disagrees, hence why it did it like this automatically haha. Maybe I can change this option somewhere so that it always uses brackets 🤔 In the meantime, will fix :)